implementations/rust/ockam/ockam_command/tests/bats/unit.bats

```diff
@@ -341,8 +341,8 @@ teardown() {
   assert_output --regexp "To Outlet Address: /service/.*/service/outlet"
 }
 
-@test "portals - delete a tcp outlet" {
-  port=5000
+@test "portals - tcp outlet CRUD" {
+  port=9000
```

Comment:
This port is used in another test. You could use 6002, 6003 and 6004 in your tests. I'll try to come up with something to avoid manually picking a port. Sorry about this 🙏

Reply:
Changed in latest commit - but I did notice a handful of other tests are sharing ports atm, notably:
- `tcp-connection CRUD`
- `tcp - create a tcp connection and then delete it`
- `portals - create an inlet/outlet pair and move tcp traffic through it` 
- `portals - create an inlet/outlet pair with relay through a forwarder and move tcp traffic through it`
(All these share port 5000)

>I'll try to come up with something to avoid manually picking a port

I could maybe implement this way to manually pick a port via a shell function at the start of whichever test has to use a port or change those ports up in the meantime in my next PR (https://github.com/build-trust/ockam/pull/4446), if you'd like.

Reply:
The port 5000 is used by the server, and that's fine for now, since we only have one server up for all the tests. The problem is sharing ports among listeners.

> I could maybe implement a way to incrementally pick a port via a shell function

The problem with that is synchronization between parallel tests, isn't it? Unless there is a way to achieve that in bash 😄